libraries/AP_SBusOut/AP_SBusOut.cpp

```diff
@@ -171,7 +171,7 @@ void AP_SBusOut::init() {
     uint16_t rate = sbus_rate.get();
 
 #if SBUS_DEBUG
-    hal.console->printf("AP_SBusOut: init %d Hz\n", rate);
+    ("AP_SBusOut: init %d Hz\n", rate);
```

Comment:
```suggestion
    PRINTF("AP_SBusOut: init %d Hz\n", rate);
```

Reply:
I made a mistake.
I'll get back to it.
This is intended to output when SBUS_DEBUG is enabled.